### PR TITLE
Fix some type of CHANGELOG-1.11.md

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
Fix some type of CHANGELOG-1.11.md
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
ping @bgrant0607 
thanks very much!

File: CHANGELOG-1.11.md [hunk 1]
```diff
@@ -1235,7 +1235,7 @@ filename | sha256 hash
 * Fixes issue for readOnly subpath mounts for SELinux systems and when the volume mountPath already existed in the container image. ([#64351](https://github.com/kubernetes/kubernetes/pull/64351), [@msau42](https://github.com/msau42))
 * Add log and fs stats for Windows containers ([#62266](https://github.com/kubernetes/kubernetes/pull/62266), [@feiskyer](https://github.com/feiskyer))
 * client-go: credential exec plugins have been promoted to beta ([#64482](https://github.com/kubernetes/kubernetes/pull/64482), [@ericchiang](https://github.com/ericchiang))
-* Revert [#64364](https://github.com/kubernetes/kubernetes/pull/64364) to resurrect rescheduler. More info https://github.com/kubernetes/kubernetes/issues/64725 :) ([#64592](https://github.com/kubernetes/kubernetes/pull/64592), [@ravisantoshgudimetla](https://github.com/ravisantoshgudimetla))
+* Revert [#64364](https://github.com/kubernetes/kubernetes/pull/64364) to resurrect rescheduler. More info https://github.com/kubernetes/kubernetes/issues/64725([#64592](https://github.com/kubernetes/kubernetes/pull/64592), [@ravisantoshgudimetla](https://github.com/ravisantoshgudimetla))
```
Comment (jbeda) [inline CHANGELOG-1.11.md]: This one still looks a little mis-formatted.  I assume it should say:

```
* Revert [#64364](https://github.com/kubernetes/kubernetes/pull/64364) to resurrect rescheduler. For more info see [#64725](https://github.com/kubernetes/kubernetes/issues/64725) and [#64592](https://github.com/kubernetes/kubernetes/pull/64592), [@ravisantoshgudimetla](https://github.com/ravisantoshgudimetla))
```

**Comment by jbeda** [comment]
Looks good except for that comment.  If I'm slow to respond feel free to ping me on slack.

File: CHANGELOG-1.11.md (hunk 1 above)
Comment (jianglingxia) [reply CHANGELOG-1.11.md]: ok，i have fixed it ,for your review,thanks

**Pull request merged**